pkg/images/checkImages.go

```diff
@@ -40,10 +40,7 @@ func CheckImages() error {
 	imagesOutsideNamespace := checkPattern(imageTagMap)
 
 	// Get a token to access the Docker Hub API
-	token, err := retrieveToken()
-	if err != nil {
-		logrus.Infof("failed to retrieve token, requests will be unauthenticated: %v", err)
-	}
+	token := getTokenFromEIO()
```

Comment:
I don't think this would work, the token still needs to be generated using the credentials

Reply:
I was assuming:
```
          secrets: |
            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKERHUB_USERNAME;
            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKERHUB_TOKEN;
```

This would get the token since it is called dockerhub_token. 
Maybe that is just the password and we still have to get the token right? 

@tashima42

Reply:
Yes, it's just the password

Reply:
Ok thanks!